images/scripts/arch.setup

```diff
@@ -106,6 +106,33 @@ echo "%wheel ALL=(ALL) ALL" > /etc/sudoers.d/90-cockpit-wheel
 useradd -c Builder builder
 echo "builder ALL = NOPASSWD: /usr/sbin/makechrootpkg, /usr/sbin/mkarchroot, /usr/bin/*-x86_64-build" > /etc/sudoers.d/90-archbuild
 
+BUILD_DEPS="\
```

Comment:
Is there some way to tell it "install all build dependencies of cockpit"? Then we'd only have a small (possilbly empty) list of extra packages. By far not a blocker though, it doesn't change often.

Reply:
We actually do record the makedepends now in the pacman database. The sync database (db pulled from the mirrors) is a simple tarball with packagename-$version/desc where `desc` is a simple file describing the package:

```
%FILENAME%
zstd-1.5.2-3-x86_64.pkg.tar.zst

%NAME%
zstd

%BASE%
zstd

%VERSION%
1.5.2-3

%DESC%
Zstandard - Fast real-time compression algorithm

%CSIZE%
671794

%ISIZE%
2487249

%MAKEDEPENDS%
gtest
```

Reply:
Raised questions for the pacman team, it would be nice to have something like apt build-deps has.

Reply:
So, I'll need to extend https://gitlab.archlinux.org/pacman/pacman/-/blob/master/src/pacman/util.c#L1145 so pacman can print the makedepends as well. Currently this is limited to a few formatting strings.